README.md

```diff
@@ -223,26 +224,39 @@ The following special character strings are supported:
 > combinations as different operating systems function differently in this
 > regard.
 
-An example of an usage with a selection range:
+Examples of usage with a selection range and `{space}` on button:
 
 ```jsx
-import React from 'react'
-import {render, screen} from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
-
-test('delete characters within the selectedRange', () => {
-  render(
-    <div>
-      <label htmlFor="my-input">Example:</label>
-      <input id="my-input" type="text" value="This is a bad example" />
-    </div>,
-  )
-  const input = screen.getByLabelText(/example/i)
-  input.setSelectionRange(10, 13)
-  userEvent.type(input, '{backspace}good')
-
-  expect(input).toHaveValue('This is a good example')
-})
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+describe('Special characters examples', () => {
+  test('delete characters within the selectedRange', () => {
+    render(
+      <div>
+        <label htmlFor="my-input">Example:</label>
+        <input id="my-input" type="text" value="This is a bad example" />
+      </div>
+    );
+    const input = screen.getByLabelText(/example/i);
+    input.setSelectionRange(10, 13);
+    userEvent.type(input, '{backspace}good');
+
+    expect(input).toHaveValue('This is a good example;');
+  });
+
+  test('emit click on button for {space}', () => {
+    const handleClick = jest.fn();
+    const { getByRole } = render(<button onClick={handleClick}>foo</button>);
+    const button = getByRole('button');
+
+    button.focus();
+    userEvent.type(button, '{space}', { skipClick: true });
```

Comment:
doesn't it seem more fitting to recommend `userEvent.keyboard('[Space]')` for this?  

`type` says: 

> Writes text inside an <input> or a <textarea>.

`keyboard`:

> Simulates the keyboard events described by text. This is similar to userEvent.type() but without any clicking or changing the selection range.

Reply:
Hi Bozdoz! Yes, you are right!
This PR is no longer needed since user-event introduced `keyboard` in v13 (March 2021) and this PR was originally created by the end of 2020...

With this PR, I tried to show an example of how to solve a particular case when pressing a button... [example and explanation](https://codesandbox.io/s/user-event-example-using-space-with-a-button-jomdg?file=/src/App.spec.js).

Anyway, thank you for your comment!